change/@fluentui-react-timepicker-compat-preview-8dca177c-555d-4973-8aa1-5ea98786788c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Deprecate preview package. Use @fluentui/react-timepicker-compat instead.",
```

Comment:
how is this being deprecated besides this message appearing in changelog after release ? 

not sure which path are we following were ( proper oss deprecation / simple deprecation as only 1 user we are in contact with is using it  ) ?

if the former,  this needs to be `npm deprecate` with on npm and public API should be marked via `@deprecated` jsdoc annotation

Reply:
I'll add the deprecate annotation. As for running npm deprecate. How can I achieve this? I don't think I have the right to do that. Is there something I can do to trigger it on pipeline?

Reply:
And the path we're going for is simple deprecation. I don't think any partner is currently using it.